account_move_line_tax_editable/models/account_move_line.py

```diff
@@ -16,3 +16,24 @@ class AccountMoveLine(models.Model):
     def _compute_is_tax_editable(self):
         for rec in self:
             rec.is_tax_editable = rec.move_id.state == "draft"
+
+    force_tax_line_id = fields.Many2one(
+        "account.tax",
+        string="Originator tax",
+        ondelete="restrict",
+        help="Indicates that this journal item is a tax line",
+        readonly=False,
+    )
+
+    @api.depends(
+        "tax_repartition_line_id.invoice_tax_id",
+        "tax_repartition_line_id.refund_tax_id",
+        "force_tax_line_id",
+    )
+    def _compute_tax_line_id(self):
+        for rec in self:
+            if rec.force_tax_line_id:
+                rec.tax_line_id = rec.force_tax_line_id
+
+        else:
+            super(AccountMoveLine, self)._compute_tax_line_id()
```

Comment:
This indentation seems incorrect, and anyway that won't work as expected:

try something like this:
```python
forced_lines = self.filtered(lambda r: r.force_tax_line_id)
for rec in forced_lines:
    rec.tax_line_id = rec.force_tax_line_id
return super(AccountMoveLine, self - forced_lines)._compute_tax_line_id()

```